public/css/colors.css

```diff
@@ -1,4 +1,7 @@
 :root {
     --default-layout-background:black;
+    --default-layout-white:#ffffff;
```

Comment:
more meaningful name layout-white did not signify anything , layout-light-background maybe